code/modules/random_map/noise/noise.dm

```diff
@@ -127,32 +126,30 @@
 			var/cellone = map[TRANSLATE_COORD(1, y)]
 			var/celltwo = map[TRANSLATE_COORD(2, y)]
 			total = cellone + celltwo
-			next_map[TRANSLATE_COORD(1, y)] = round(total / 2)
+			map[TRANSLATE_COORD(1, y)] = round(total / 2)
 			// hardcoding x=2 also, to lower checks in the loop
 			// larger radius would need to also cover all x < 1 + blur_radius
 			total += map[TRANSLATE_COORD(3, y)]
-			next_map[TRANSLATE_COORD(2, y)] = round(total / 3)
+			map[TRANSLATE_COORD(2, y)] = round(total / 3)
 			for(var/x = 3 to limit_x-1) // should technically be 2 + blur_radius to limit_x - blur_radius
 				total -= map[TRANSLATE_COORD(x-2, y)] // x - blur_radius - 1
 				total += map[TRANSLATE_COORD(x+1, y)] // x + blur_radius
-				next_map[TRANSLATE_COORD(x, y)] = round(total / 3) // should technically be 2*blur_radius+1
+				map[TRANSLATE_COORD(x, y)] = round(total / 3) // should technically be 2*blur_radius+1
 		// now do the same in the x axis
-		map = next_map.Copy()
```

Comment:
i believe this is not actually the case because of the specific box blur we use, it adds the unblurred... oh you're right, damn. ok i'll drop the commit, it was negligible fractions of a second anyway